pkg/cluster/internal/delete/delete.go

```diff
@@ -46,5 +46,10 @@ func Cluster(c *context.Context) error {
 		fmt.Printf("$KUBECONFIG is still set to use %s even though that file has been deleted, remember to unset it\n", c.KubeConfigPath())
 	}
 
-	return nodes.Delete(n...)
+	err = nodes.Delete(n...)
+	if err != nil {
```

Comment:
we should probably collect this error but still try to delete the network

Reply:
the reason I originally wrote this is because if the node delete fails, it is very likely that its state is still active (Up). 
then deleting the network will also fail because `docker network rm` does not allow the removal of network with active endpoints.

Reply:
the error can be scoped:
```
if err := nodes.Delete(n...); err != nil {
...
}
```